Gemfile

```diff
@@ -1,19 +1,20 @@
 source "https://rubygems.org"
 
-gem 'bundler'
+gem "bundler"
 
 gemspec
 
-gem 'coveralls', require: false
+gem "coveralls", require: false
+gem "standard"
```

Comment:
Honest question: how do you use bundle groups for gem development? 

I tend to think of gems as having only direct dependencies and development dependencies. Any gem in the Gemfile is effectively a "non-direct" dependency. I could imagine a use case, maybe CI?, where one to filter out a certain subset of dependencies but I couldn't find a place where this gem might be doing that now.

For app development, the use cases are more clear since frameworks (in other words, Rails) bake in the notion of separate environments.

Reply:
Honest answer, I have no clue! My Ruby background is primarily from working on Rails SaaS apps so I'm not sure about library development requirements/best practices. 

Thinking about this a bit more deeply, I suppose it's the [ruby-oembed.gemspec](https://github.com/ruby-oembed/ruby-oembed/blob/master/ruby-oembed.gemspec) file that specifies the peer dependencies that apps consuming this gem will be installing.

Meanwhile the [Gemfile](https://github.com/ruby-oembed/ruby-oembed/blob/master/Gemfile) I suppose is only used for local development.

So if anything I'd lean towards removing the groups from the Gemfile (since they don't currently appear to be in use) in an attempt to avoid future confusion.